tfc-dump.pl

```diff
@@ -156,14 +166,11 @@
 }
 
 # Get the number of Variable Sets
-
-$jq_cmd      = "cat $tmpfile | jq '.meta.pagination[\"total-count\"]'";
-$total_count = `$jq_cmd`;
-unlink($tmpfile);
-
-if ($total_count > $pg_size) {
-	print STDERR "WARNING: ${total_count}-${pg_size} Variable Sets were not backed up.\n";
-	exit(1);
-}
-
+# $jq_cmd      = "cat $tmpfile | jq '.meta.pagination[\"total-count\"]'";
```

Comment:
I think that running through multiple pages should generate a total_of_everything_count which could be used in similar revised logic. Removing this code instead of revising it is not wise. Error checking and feedback are always good.

Reply:
A revision to this logic is my second requirement.